tests/models/test_amp.py

```diff
@@ -10,17 +10,75 @@
 from tests.base import EvalModelTemplate
 
 
-@pytest.mark.spawn
-@pytest.mark.parametrize("backend", ['dp', 'ddp'])
 @pytest.mark.skipif(not torch.cuda.is_available(), reason="test requires GPU machine")
-def test_amp_single_gpu(tmpdir, backend):
+def test_amp_single_gpu_dp(tmpdir):
```

Comment:
there is no reason to split it just correct the backend
```
@pytest.mark.parametrize("backend", ['dp', 'ddp_spawn'])
```

Reply:
it doesn't work.... i mentioned this on slack. it's a weird pytest problem.